### DISCUSSION TITLE
Trying to contact Shahin sherman89 about his WpfReporting

### DISCUSSION THREAD

**Discussion by msiminski** [community/community#35368] [Other Feature Feedback, Questions, & Ideas] [answered]
I am trying to contact Shahin sherman89.  He has a WpfReporting project and I am using it and it works great for a single list of information and paging but I am trying to use it with several lists together.  Not sure if I am putting them in the XAML right or if I need to do each list separate as an xps doc and add the pages together somehow.  I am new to XPS Documents and this may be all my issue but I would love to know that.

**Comment by airtower-luna** [comment, marked as answer]
I assume you're looking for help with https://github.com/sherman89/WpfReporting? In that case you should open an issue *in that repository* so the owner has a better chance to see it. However there hasn't been publicly visible activity for a while, so I don't know if they'll pay attention. :sweat_smile:
